feature-libs/quote/components/header/overview/quote-header-overview.component.ts

```diff
@@ -0,0 +1,257 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, inject } from '@angular/core';
+import { EventService, TranslationService } from '@spartacus/core';
+import {
+  Quote,
+  QuoteAction,
+  QuoteActionType,
+  QuoteFacade,
+  QuoteMetadata,
+  QuoteState,
+} from '@spartacus/quote/root';
+import { Card, ICON_TYPE } from '@spartacus/storefront';
+import { combineLatest, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { QuoteUIConfig } from '../../config/quote-ui.config';
+import {
+  EditCard,
+  SaveEvent,
+} from '../buyer-edit/quote-header-buyer-edit.component';
+
+@Component({
+  selector: 'cx-quote-header-overview',
+  templateUrl: './quote-header-overview.component.html',
+})
+export class QuoteHeaderOverviewComponent {
+  protected quoteFacade = inject(QuoteFacade);
+  protected eventService = inject(EventService);
+  protected translationService = inject(TranslationService);
+  protected config = inject(QuoteUIConfig);
```

Comment:
@Larisa-Staroverova 
```suggestion
  protected quoteUIConfig = inject(QuoteUIConfig);
```

Reply:
@ChristophHi, there are several places where this "issue" appears.
I will fix all of them accordingly.

Reply:
Done!